includes/class-rest.php

```
@@ -433,8 +440,32 @@ public function database_cleanup( \WP_REST_Request $request ) {
+				$failures = array();
+
+				foreach ( $results as $key => $value ) {
+					if ( is_wp_error( $value ) ) {
+						$failures[ $key ] = $value->get_error_message();
+					} else {
+						$total += (int) $value;
+					}
+				}
+
+				if ( ! empty( $failures ) ) {
+					return new \WP_REST_Response(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '456', column '17'). (undefined)

(MissingImport)